index.test.js

```diff
@@ -604,16 +604,37 @@ test('clears empty selector after comma', () => {
   run('a, b { .one, .two, {} }', 'a .one, a .two, b .one, b .two {}')
 })
 
-test('moves comment with rule', () => {
-  run('a { /*B*/ /*B2*/ b {} }', '/*B*/ /*B2*/ a b {}')
+test("Save the parent's comment", () => {
+  run('a { /*i*/ b {} }', 'a { /*i*/ } a b {}')
 })
 
-test('moves comment with at-rule', () => {
-  run('a { /*B*/ @media { one: 1 } }', '/*B*/ @media {a { one: 1 } }')
+test('Save the comments for the parent and child', () => {
```

Comment:
Can we add test with `\n` like:

```css
a { 
    /*i*/ 

    b {} }
```

Seems like often you will not have inside-commend.

Reply:
We can add it. I've already checked, it will work properly.

Reply:
Yes, let's add it to not break in the future